src/main/java/com/leetcode/practice/easy/binary_search/BinarySearch.java

```diff
@@ -0,0 +1,38 @@
+package com.leetcode.practice.easy.binary_search;
+
+import java.util.Arrays;
+
+public class BinarySearch {
+
+    public boolean isTargetNumberPresent(int[] arr, int target) {
```

Comment:
isTargetNumberPresent will make it tightly coupled with Numbers. What if you want to overload this method for string, characters etc.
So choose a name that can be easily overloaded and serve the same purpose. Like isTargetPresent